### ISSUE TITLE
LinkedinOAuth deprecated?

### ISSUE THREAD

**Issue by foxale** [python-social-auth/social-examples#5]
**Steps to reproduce:** Clone the repository and try running _example_flask_

> AttributeError: module 'social_core.backends.linkedin' has no attribute 'LinkedinOAuth'

I guess it's a result of this one: https://github.com/python-social-auth/social-core/issues/310

**Comment by gersona** [comment]
That's correct, `social_core.backends.linkedin.LinkedinOAuth` is no longer valid and should be removed from `AUTHENTICATION_BACKENDS` 
- Same for `social_core.backends.yahoo.YahooOpenId` as of https://github.com/python-social-auth/social-core/pull/523
